src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
```

Comment:
The class name `FruitTransaction` is not fully descriptive of the contents within the class. Consider renaming the class to more accurately reflect that it contains a transaction type (operation), fruit name, and quantity. For example, `FruitOperation` might be a more descriptive name.